views/watchlist_funcs.php

```diff
@@ -1,41 +1,31 @@
  <?php
+  require_once("../database/setup.php");
 
   // Follow or unfollow an item ()
-  if (!isset($_POST['functionname']) || !isset($_POST['arguments'])) {
+  if (!isset($_POST['functionname']) || !isset($_POST['argument_1']) || !isset($_POST['argument_2']) || !isset($_POST['argument_3'])) {
     return;
   }
+  // Extract variables from the POST variables (follower_rows exists to check that the user isn't already on the watchlist)
+  $item_id = $_POST['argument_1'];

```

Comment:
It a bad practice to call things "argument_1", it would be much better to just change the names of what you are passing to the post with something more telling

Reply:
Yup, good point, changed it